docs/sources/setup-grafana/installation/kubernetes.md

```diff
@@ -241,14 +252,15 @@ spec:
   sessionAffinity: None
   type: LoadBalancer
 ```
+> **Note:** Using LoadBalancer in the Service may expose your Grafana instance to the Internet depending on your cloud platform and network configuration. Instead, you can use ClusterIP to restrict access from within the cluster Grafana is deployed to.
 
 1. Send manifest to Kubernetes API Server
    `kubectl apply -f grafana.yaml`
 
-1. Check that it worked by running the following:
+2. Check that it worked by running the following:
```

Comment:
Are these numbered because the numbering does not render correctly as `1.`, `1.`, `1.`? If so, leave as is; if not, revert such changes.

Reply:
By default for numbered lists, we use a series of `1.`s.

Reply:
That's because VS Code automatically changes numbers. I have to find a way to turn that off.

Reply:
Updated and turned off auto numbering.